manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -1,10 +1,17 @@
 linux:
   enabled: true
+
+  # Pod security context
+  podSecurityContext: {}
+
   image:
     repository: registry.k8s.io/csi-secrets-store/driver
     tag: v1.4.1
     #digest: sha256:
     pullPolicy: IfNotPresent
+  # The secrets-store container security context
+  securityContext:
+    privileged: true
```

Comment:
`privileged: true` is required for the driver to function correctly. Moving this to `values.yaml` could cause the driver functionality to fail if not set correctly by the user.

`privileged: true` should instead remain in the manifest and this field should append to the existing `securityContext`.

Reply:
Thanks for suggestion. Done.